src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -19,36 +21,34 @@
   [view-id]
   (key-value-event "navigation" :viewId view-id))
 
+(defn screen-event
+  [screen event-data]
+  (let [screen-id (:name screen)
+        event-id  (get-in screen [:metrics :event :id] screen-id)]
+    {:metric
+     {:eventName  "navigation"
+      :platform   platform/os
+      :appVersion build/app-short-version
+      :eventValue (assoc event-data
+                         :viewId          (name screen-id)
+                         :metric-id-alias (name event-id)
+                         :screen-id       (-> screen-id symbol str))}}))
```

Comment:
what is the difference between `view-id` and `screen-id`. Is there any value in tracking both?

Reply:
`:screen-id` (recently renamed to `:screen-name`) is the string formatted like this `screen/profile.settings`. And `viewId` would be the string `profile.settings`. `screen-id` is mostly here to give an exact match on the keyword used for identifying the screen.

Reply:
do we need both? Just wondering if we're not sending redundant data.

Reply:
Having both is likely redundant, and I would be in favour of sending one of them, I think the value for `screen-id` is more accurate, but I've left both for now for backwards compatibility reasons.

If we get green-light to use the screen-id value, then we can remove the other one, but for the moment we're extending the data to avoid a breaking change.

Reply:
thanks for clearing it up :+1: